crates/ibc/src/core/ics03_connection/handler/conn_open_try.rs

```diff
@@ -215,15 +221,15 @@ mod tests {
                 want_pass: false,
             },
             Test {
-                name: "Good parameters but has previous_connection_id".to_string(),
+                name: "Good parameters (no previous_connection_id)".to_string(),
                 ctx: context.clone().with_client(&msg_conn_try.client_id, Height::new(0, client_consensus_state_height).unwrap()),
                 msg: ConnectionMsg::ConnectionOpenTry(Box::new(msg_conn_try.clone())),
-                want_pass: false,
+                want_pass: true,
             },
             Test {
                 name: "Good parameters".to_string(),
                 ctx: context.with_client(&msg_conn_try.client_id, Height::new(0, client_consensus_state_height).unwrap()),
-                msg: ConnectionMsg::ConnectionOpenTry(Box::new(msg_conn_try.with_previous_connection_id(None))),
```

Comment:
I think we can remove `with_previous_connection_id()` now.

Reply:
[253780a](https://github.com/cosmos/ibc-rs/pull/170/commits/253780a8fd54db6e174ecb1848aa72ab94cd84fd)